nginx_rock/rockcraft.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: synapse-nginx
+summary: synapse nginx rock
+description: Nginx OCI image for the synapse charm
+version: "1.0"
+base: ubuntu:22.04
+build-base: ubuntu:22.04
+license: Apache-2.0
+platforms:
+  amd64:
+parts:
+  add-user:
+    plugin: nil
+    overlay-script: |
```

Comment:
Now there's the possibility of using the user rocks provides out of the box

Reply:
Related: #51